live/engine/live_exec.py

```
@@ -2723,6 +2723,203 @@ def _usd(v):
+            try:
+                committed += float(op.get("initialValue") or 0.0)
+                unrealized += float(op.get("cashPnl") or 0.0)
+                cur_values.append(float(op.get("currentValue") or 0.0))
+            except (TypeError, ValueError):
+                continue
+        # Naked = committed capital not paired by the smaller leg. This is a
+        # conservative venue-truthful proxy; the bot own float_marks remain
+        # the authoritative source for the intraday chart.
+        try:
+            naked = max(0.0, committed - (min(cur_values) if cur_values else committed))
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)